### PR TITLE
retrieval code

### PR DESCRIPTION
**Pull request by calee14**

### CODE REVIEW

File: opportunity/dbmodels/oppmodel.py [hunk 1]
```diff
@@ -34,5 +34,17 @@ def add(self, opp_title, date, img_url, desc, link, topic, author):
...
+    def load_recent_posts(self, num=1):
+        # returns a cursor but data can be accessed through a for loop or through indices
+        # load the most recent posts going up to num
+        return self.db.opps.find().sort("date", -1).limit(num)
+
+    def load_spliced(self, skip=0, batch_size=5):
+        test = self.db.opps.find().sort("date", -1).limit(batch_size).skip(skip)
+        for i in test:
+            print(i)
+        return 'as'
```
Comment (RafaelCenzano) [inline opportunity/dbmodels/oppmodel.py:47]: this doesn't return the right thing, does it?

File: opportunity/dbmodels/oppmodel.py [hunk 2]
```diff
@@ -34,5 +34,17 @@ def add(self, opp_title, date, img_url, desc, link, topic, author):
             return True
         else:
             return False
+
+    def load_recent_posts(self, num=1):
+        # returns a cursor but data can be accessed through a for loop or through indices
+        # load the most recent posts going up to num
+        return self.db.opps.find().sort("date", -1).limit(num)
+
```
Comment (RafaelCenzano) [inline opportunity/dbmodels/oppmodel.py:42]: can't these be one method?

File: opportunity/views.py [hunk 3]
```diff
@@ -18,12 +19,11 @@ def load_user(id):
...
 @app.route('/', methods=['GET'])
-def index():
-    print(current_user)
-    #opp = Opportunity()
-    #print(opp.add("opp_title","date", "img", "desc", "link", "topic", "author"))
-
-    return render_template('home.html')
+def index(): 
+    opp = Opportunity()
+    opp.add("opp_title",datetime.now(), "img", "desc", "link", "topic", "author")
```
Comment (RafaelCenzano) [inline opportunity/views.py:24]: should we use utc or something for time?

**Review by RafaelCenzano — APPROVED** [review]

**Comment by calee14** [comment]
also fixed user registration

**Pull request merged**